config-model/src/main/java/com/yahoo/vespa/model/admin/monitoring/VespaMetricSet.java

```diff
@@ -176,21 +176,19 @@ private static Set<Metric> getContainerMetrics() {
         addMetric(metrics, ContainerMetrics.HTTPAPI_CONDITION_NOT_MET.baseName(), List.of("rate"));
         addMetric(metrics, ContainerMetrics.HTTPAPI_NOT_FOUND.baseName(), List.of("rate"));
 
-        metrics.add(new Metric("mem.heap.total.average"));
-        metrics.add(new Metric("mem.heap.free.average"));
-        metrics.add(new Metric("mem.heap.used.average"));
-        metrics.add(new Metric("mem.heap.used.max"));
+        addMetric(metrics, ContainerMetrics.MEM_HEAP_TOTAL.baseName(), List.of("average"));
```

Comment:
consider using private static void addMetric(Set<Metric> metrics, String nameWithSuffix) instead of list with one item (applies to the other below, too, with only one suffix)

Reply:
Thanks. Changed. I agree this is better for readability and also prevents possible typos from being merged.